README.md

```diff
@@ -148,7 +148,10 @@ FreeBSD etc). However, mlpack is in the repositories of many Linux distributions
 and so it may be easier to use the package manager for your system.  For example, 
 on Ubuntu, you can install mlpack with the following command:
 
-    $ sudo apt-get install libmlpack-dev
+    $ sudo apt-get install mlpack-bin
+
+On Fedora or Red Hat:
+    $ sudo dnf install mlpack
```

Comment:
We might need `mlpack-devel` and `mlpack-bin` too to match the Ubuntu command. `mlpack-bin` depends on `mlpack`, so we can just have `mlpack-bin`.

Reply:
But @rcurtin are mlpack or related headers and binaries their in latest red hat repositories. It's not their in Red hat 8.3.1 I believe.

Reply:
Yeah, for RHEL it's in the EPEL repositories.  You could just write `In Fedora or Red Hat (EPEL):`.